src/main/java/org/fundacionjala/coding/yerel/ExcerciseCountChar.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.coding.yerel;
+
+/**
+ * this class count cuantity of leter in sentense.
+ */
+public class ExcerciseCountChar {
+
+    /**
+     * @param sentence is a sentecense to count.
+     * @param leter    it is a specific leter find.
+     * @return a number cuantity.
+     */
+    public int countChar(String sentence, String leter) {
+        int count = 0;
+        for (int i = 0; i < sentence.length(); i++) {
+            if (String.valueOf(sentence.charAt(i)).toLowerCase().equals(leter)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)